### PR TITLE
STL-802: Changed state methods from get and set to get_state and set_…

### PR DESCRIPTION
**Pull request by BhamidiK**
…state in C++, and getState and setState in java interfaces. Updated the tests

Signed-off-by: Bhamidi Krishna<bhamidik@hotmail.com>

### CODE REVIEW

**Comment by peterschwarz** [comment]
Thanks for the PR.  There are couple things that need to get cleaned up here, with respect to the commit process.  
1) Could you fixup the commit message to follow the rules we reference in the contribution guide (found [here](https://chris.beams.io/posts/git-commit/#seven-rules))
2) Make sure you sign off with the DCO (see under the list of Checks)

Full commit process is found here:
https://sawtooth.hyperledger.org/docs/core/releases/latest/community/contributing.html#commit-process

File: sdk/cxx/include/sawtooth/global_state.h [hunk 1]
```diff
@@ -40,31 +40,31 @@ class GlobalState {
     // multiple values, it is encouraged to use the batch get overload defined
     // below. The function returns false if the value is not retrieved, true if
     // it is found in the GlobalStore, and throws an exception if an error occurs
-    bool Get(std::string* out_value, const std::string& address) const;
+    bool get_state(std::string* out_value, const std::string& address) const;
```
Comment (peterschwarz) [inline sdk/cxx/include/sawtooth/global_state.h]: The casing should remain the same for this library:  GetState, SetState, DeleteState

**Comment by peterschwarz** [comment]
Looks like there might also be an issue with the way that you did the sign-off in your commit.  Again, take a look at the guide I posted in an earlier comment here.

**Comment by ineffectualproperty** [comment]
@BhamidiK I believe your e-mail address needs to be in < > to be captured by the DCO bot, e.g. <bhamidik@hotmail.com>. The e-mail may also need to match your github account.

**Comment by BhamidiK** [comment]
Please can you kick-off a build? My builds fail: 
_[jenkins-Sawtooth-Hyperledger-sawtooth-core-PR-1071-4] Running shell script
+ ./bin/whitelist BhamidiK COMMITTERS
FAILED: User 'BhamidiK' not whitelisted._

**Comment by peterschwarz** [comment]
It looks like this PR might have had a messy rebase or merge.  Could you close it and resubmit a new PR with a single commit?

**Comment by BhamidiK** [comment]
Sure, will do.


Krishna


________________________________

**Pull request closed without merging**